core/go/internal/domainmgr/private_smart_contract.go

```diff
@@ -250,11 +252,14 @@ func (dc *domainContract) AssembleTransaction(dCtx components.DomainContext, rea
 		// - Attestation plan identities
 		// - State distributions
 		// - Associated nullifier requests
+		log.L(dCtx.Ctx()).Info("Qualifying assembly identities")
 		dc.fullyQualifyAssemblyIdentities(res)
 
 		// Note the states at this point are just potential states - depending on the analysis
 		// of the result, and the locking on the input states, the engine might decide to
 		// abandon this attempt and just re-assemble later.
+		log.L(dCtx.Ctx()).Debugf("Number of potential output states: %+v", len(res.AssembledTransaction.OutputStates))
```

Comment:
These log entries feel like intermediate log entries from code dev, which need to be consolidated into a log entry from this function that can be left on.

Reply:
I've raised https://github.com/LFDT-Paladin/paladin/issues/890 on the V1 project board to make sure we don't miss the job of reviewing logs

Reply:
Will be a non-trivial, but important item, to really validate what Info/Debug/Trace should be with real scenarios. Let's not consider it an optional path on the V1 journey.

If you found turning something into trace mode too hard, and it lead to adding debug entries instead - we really *must* solve that. Because it will be a pattern all developers repeat over time.

So while it's fresh, raising a must-fix item for whichever test/env you're testing with that is hard to put into trace mode I think is just as (if not more) important than the reconciling of log entries.

Reply:
Yes understood, will add a comment to https://github.com/LFDT-Paladin/paladin/issues/890 to ensure this is factored in.